tests/network/utils.py

```diff
@@ -307,11 +318,32 @@ def assert_authentication_request(vm, service_app_name):
     request_response = authentication_request(
         vm=vm,
         service=service_app_name,
-        expected_output=expected_output,
     )
     assert_service_mesh_request(expected_output=expected_output, request_response=request_response)
 
 
+def exec_console_command(vm, command, timeout=TIMEOUT_1MIN):
+    """
+    Execute a command in VM console and return the output.
+
+    Args:
+        vm (VirtualMachine): VM to be used for console connection.
+        command (str): Command to execute.
+        timeout (int, default=TIMEOUT_1MIN): Timeout for the command execution.
+
+    Returns:
+        str: Command output (child.before.decode('utf-8')).
+    """
+    with console.Console(vm=vm) as vmc:
+        LOGGER.info(f"Execute {command} on {vm.name}")
+        try:
+            vmc.sendline(command)
+            vmc.expect([r"\$"], timeout=timeout)
+            return vmc.before
+        except (pexpect.exceptions.TIMEOUT, pexpect.exceptions.EOF):
+            return vmc.before
```

Comment:
Logging fits here, so separating may be useful.

Reply:
Ack, thanks.